src/app/frontend/index_module.js

```diff
@@ -54,6 +54,7 @@ export default angular
     .module(
         'kubernetesDashboard',
         [
+          'angular-clipboard',
```

Comment:
Can't this dependency be added to resource module directly? It's used only there. We should clean up our deps eventually :)